static/src/javascripts/projects/common/modules/discussion/comment-count-widget.js

```diff
@@ -0,0 +1,79 @@
+/* globals guardian */
+define([
+    'comment-count',
+    'common/utils/get-property',
+    'fastdom',
+    'common/modules/experiments/ab',
+    'common/modules/experiments/tests/utils/comment-blocker',
+    'common/utils/assign',
+    'common/utils/formatters',
+    'common/utils/mediator',
+    'common/utils/report-error'
+], function (
+    commentCountWidget,
```

Comment:
Am I missing something here(I'm fairly sure I am)? Isn't this file comment count widget?